signinwithapplebutton/src/main/java/com/willowtreeapps/signinwithapplebutton/view/SignInWebViewClient.kt

```diff
@@ -30,13 +31,13 @@ class SignInWebViewClient(
 
                 when {
                     codeParameter == null -> {
-                        callback.onSignInFailure(IllegalArgumentException("code not returned"))
+                        callback.onSignInWithAppleFailure(IllegalArgumentException("code not returned"))
                     }
                     stateParameter != attempt.state -> {
-                        callback.onSignInFailure(IllegalArgumentException("state does not match"))
+                        callback.onSignInWithAppleFailure(IllegalArgumentException("state does not match"))
                     }
                     else -> {
-                        callback.onSignInSuccess(codeParameter)
+                        callback.onSignInWithAppleSuccess(codeParameter)
```

Comment:
I lengthened these names to avoid namespace collisions because I expect the consumer to implement them in their Activity.